blacken_docs.py

```diff
@@ -36,6 +36,13 @@
     r'(?P<after>^(?P=indent)\\end{minted}\s*$)',
     re.DOTALL | re.MULTILINE,
 )
+PYTHONTEX = '(pyblock|pycode|pyconsole|pyverbatim)'
+PYTHONTEX_RE = re.compile(
+    rf'(?P<before>^(?P<indent> *)\\begin{{{PYTHONTEX}}}\n)'
+    r'(?P<code>.*?)'
+    rf'(?P<after>^(?P=indent)\\end{{{PYTHONTEX}}}\s*$)',
+    re.DOTALL | re.MULTILINE,
+)
```

Comment:
this should probably just extend the `LATEX_RE` above instead of duplicating I would assume?